src/Export/Exporter.php

```diff
@@ -26,6 +32,51 @@
  *
  * @deprecated since sonata-project/admin-bundle 3.14, to be removed in 4.0.
  */
-class Exporter extends BaseExporter
+class Exporter
```

Comment:
Technical a BC break, but this should only affect a few people

Reply:
Why is it a BC break ? Should I merge in this master instead then ?

Reply:
Because you remove methods provided by `BaseExporter`. Also `instanceof` checks will fail.

WDYT @greg0ire @OskarStark ?

Reply:
Maybe I was tired but the only method I found was `getResponse` that's why I copy/paste it.

But indeed for `instanceof`...
If we don't drop the CoreBundle in a minor, we'll need to release SonataAdmin 4 to support Symfony5/Twig3 or make the CoreBundle compatible with SF5/Twig3.

Reply:
Perhaps it is best to separate removing CoreBundle and upgrading to Symfony5. I have started to see if SF5 compatibility can be added to CoreBundle. Shall I create a PR to see if it could work?